app/editor/package.json

```diff
@@ -34,7 +34,7 @@
     "react-tooltip": "4.2.21",
     "redux-logger": "3.0.6",
     "styled-components": "5.3.5",
-    "tno-core": "0.0.113",
+    "tno-core": "0.0.116",
```

Comment:
Would love to know how we can test updates to the package without publishing.  It's very annoying.